docs/quick-ingestion-guides/looker/configuration.md

```diff
@@ -0,0 +1,251 @@
+</p>
+
+4. Create a secret to store the API Key Client Secret
+
+  This will securely store your client secret"
+
+   * Enter a name like `LOOKER_CLIENT_SECRET` - we will use this later to refer to the client secret
+   * Enter the client secret
+   * Optionally add a description
+   * Click **Create**
+
+<p align="center">
+   <img width="70%" alt="API Key client secret" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/looker/looker-client-secret.png"/>
+</p>
+
+## Configure Recipe
+1.  Navigate to the **Sources** tab and click **Create new source**
+
+  <p align="center">
+    <img width="75%" alt="Click &quot;Create new source&quot;" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/common/common_ingestion_click_create_new_source_button.png"/>
+  </p>
+
+2.  Choose Looker
+
+  <p align="center">
+    <img width="70%" alt="Select Looker from the options" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/looker/looker-choose-looker.png"/>
+  </p>
+
+3.  Enter details into the Looker Recipe
+
+    You need to set minimum 3 field in the recipe:
+    
+    a. **Base URL:** This is your looker instance URL. For example https://abc.cloud.looker.com.
+
+    b. **Client ID:** Use the secret LOOKER_CLIENT_ID with the format `${LOOKER_CLIENT_ID}`.
+
+    c. **Client Secret:** Use the secret LOOKER_CLIENT_SECRET with the format `${LOOKER_CLIENT_SECRET}`.
+
+
+
+Optionally, use the `dashboard_pattern` and `chart_pattern` fields to filter for specific dashboard and chart.
+
+    config:
+         ...
+         dashboard_pattern:
+            allow:
+              - "2"
+         chart_pattern:
+            allow:
+              - "258829b1-82b1-4bdb-b9fb-6722c718bbd3"
+
+Your recipe should look something like this:
+<p align="center">
+  <img width="70%" alt="Looker Recipe" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/looker/looker-ingestion-source-recipe.png"/>
+</p>
+
+
+After completing the recipe, click **Next**.    
+
+## Schedule Execution
+
+Now it's time to schedule a recurring ingestion pipeline to regularly extract metadata from your Looker instance.
+
+1. Decide how regularly you want this ingestion to run-- day, month, year, hour, minute, etc. Select from the dropdown
+
+<p align="center">
+    <img width="75%" alt="schedule selector" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/common/common_ingestion_set_execution_schedule.png"/>
+</p>  
+
+2. Ensure you've configured your correct timezone
+<p align="center">
+    <img width="75%" alt="timezone_selector" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/common/common_ingestion_set_execution_timezone.png"/>
+</p>  
+
+3. Click **Next** when you are done
+
+## Finish Up
+
+1. Name your ingestion source, then click **Save and Run**
```

Comment:
```suggestion
Name your ingestion source, then click **Save and Run**
```

I don't think we need numbering here 
<img width="497" alt="image" src="https://github.com/datahub-project/datahub/assets/26561086/562de834-b465-4e3f-a261-8e39cfca22ca">

Reply:
done